docs/enhancements/0001_project_enhancements.md

```diff
@@ -0,0 +1,103 @@
+# 0001: Project process refinement for 2023
```

Comment:
missed the "timeline". If this is really meant as our wider 2023 goal, should we propose some checkpoints? Like put it on the agenda every quarter, check where we are and refine our plan?

Reply:
ack, will add 👍